exercises/collatz-conjecture/src/example/kotlin/CollatzCalculator.kt

```diff
@@ -0,0 +1,12 @@
+object CollatzCalculator {
```

Comment:
I'm not sure if there's a general consensus on this, but most of the time I would pick an object-scope function unless the solution needs some state, in which case I'd make a class.